padre_meddea/calibration/calibration.py

```diff
@@ -22,6 +22,11 @@
     "process_file",
     "get_calibration_file",
     "read_calibration_file",
+    
+    "spectrum",
+    "calibrate_spectrum",
+    "nearest_energy",
```

Comment:
This update breaks the module because you are trying to import a function that you have not defined.